remhos.cpp

```diff
@@ -1349,6 +1415,35 @@ void AdvectionOperator::Mult(const Vector &X, Vector &Y) const
    }
 }
 
+double AdvectionOperator::ComputeTimeStepEstimate(const Vector &x,
+                                                  const Vector &dx,
+                                                  const Vector &x_min,
+                                                  const Vector &x_max) const
+{
+   if (dt_control == TimeStepControl::FixedTimeStep) { return dt; }
+
+   // x_min <= x + dt * dx <= x_max.
+   int n = x.Size();
+   const double cfl = 0.5, eps = 1e-15, dt_min = 1e-10;
```

Comment:
perhaps make eps=1e-14, I've run into issues with 1e-15 
```suggestion
   const double cfl = 0.5, eps = 1e-14, dt_min = 1e-10;
```